File: task/src/main/java/com/dh/task/data/source/remote/TaskService.java
package com.dh.task.data.source.remote;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

/**
 * Created by Jin on 2021/2/23.
 * Description
 */
public interface TaskService {

    @GET("group/{id}/users")
    Call<List<String>> taskList(@Path("id") int taskId);
}

File: task/src/main/java/com/dh/task/data/source/remote/TaskService.kt
package com.dh.task.data.source.remote

import com.dh.task_api.entity.Task
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

/**
 * Created by Jin on 2021/2/24.
 * Description
 */
interface TaskService {

    @GET("group/{id}/users")
    fun taskList(@Path("id") taskId: Int): Call<MutableList<Task>>

    @GET("group/{id}/users")
    suspend fun taskListCoroutine(@Path("id") taskId: Int): Call<MutableList<Task>>
}